docs/concepts/deploy-url.md

```diff
@@ -0,0 +1,90 @@
+<!--
+kb_concepts
+kb_pwa
+kb_everyone
+kb_sync_latest_only
+-->
+
+# Deploy URL
```

Comment:
@marschmidt89 We have a new documentation snippet here. I'll read through it semantically. Could you please check according to your standards?